mollie/api/client.py

```diff
@@ -247,15 +252,15 @@ def setup_oauth(self, client_id, client_secret, redirect_uri, scope, token, set_
                 'client_id': client_id,
                 'client_secret': self.client_secret,
             },
-            auto_refresh_url='https://api.mollie.com/oauth2/tokens',
+            auto_refresh_url=self.OAUTH_AUTO_REFRESH_URL,
             redirect_uri=redirect_uri,
             scope=scope,
             token=token,
             token_updater=set_token
         )
         authorization_url = None
         if not self.oauth.authorized:
-            authorization_url, state = self.oauth.authorization_url('https://www.mollie.com/oauth2/authorize')
+            authorization_url, state = self.oauth.authorization_url(self.OAUTH_AUTHORIZATION_URL)
```

Comment:
The old URL is not part of the API, as it is on www.mollie.com (not api.mollie.com). Is the endpoint in the API the correct one?

Reply:
Fixed. Good catch!